src/lib/services/generative/ConversationAI.ts

```
@@ -0,0 +1,583 @@
+      case 'metric':
+        return parseFloat(value.replace(/[^0-9.-]/g, ''));
+
+      case 'location':
+      case 'state':
+        // Normalize state names to abbreviations
+        const stateMap: Record<string, string> = {
+          'new york': 'NY',
+          california: 'CA',
+          texas: 'TX',
+          // ... etc
+        };
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)